kb/2024-01-30/the_potential_risk_with_fstrim.md

```diff
@@ -12,9 +12,21 @@ hide_table_of_contents: false
 ---
 
 :::note
-This issue is already resolved by Longhorn v1.5.4, v1.6.1, v1.7.0 and later versions.
-:::
+Filesystem trim is introduced in Longhorn v1.4.0 from [Issue 836](https://github.com/longhorn/longhorn/issues/836) 
+
+Please check the following form for the impacted version.
+
 
+| Harvester Version | Longhorn Version | Impacted by Filesystem Trim |
+|-------------------|------------------|-----------------------------|
+| v1.2.0            | v1.4.3           | Yes                         | 
+| v1.2.1            | v1.4.3           | Yes                         |
+| v1.2.2            | v1.5.5           | No                          |
+| v1.3.0            | v1.6.0           | Yes                         |
+| v1.3.1 (upcoming) | v1.6.2           | No                          |
```

Comment:
I'd suggest we list affected versions only or the opposite way to show the fixed version.
@jillian-maroket do you have any suggestions?